mongodbatlas.go

```diff
@@ -74,6 +74,10 @@ func (m *MongoDBAtlas) Initialize(ctx context.Context, req dbplugin.InitializeRe
 	resp := dbplugin.InitializeResponse{
 		Config: req.Config,
 	}
+	resp.SetSupportedCredentialTypes([]dbplugin.CredentialType{
+		dbplugin.CredentialTypePassword,
+		dbplugin.CredentialTypeClientCertificate,
```

Comment:
This should be resolved by running the following:
```
go get github.com/hashicorp/vault/sdk@db-secrets-cred-type-client-cert && go mod tidy
```

The SDK in the Vault branch has changes that are needed to bring into the plugin for it to compile.